### PR TITLE
fix: ticket quantity numbers

### LINKED ISSUE #4613: Ticket Order Page shows 0 Tickets bought instead of 2 Tickets
The order pages do not seem to show the correct numbers of tickets bought. A user who bought 2 tickets has 0 tickets showing on his order page, e.g. here https://eventyay.com/orders/89e637e6-a281-41d5-9153-6b24f7defa3b/view

Screenshot from 2020-07-22 21-42-12

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5667
Fixes #4613 

https://user-images.githubusercontent.com/56407566/108152232-9ba1ac00-70fe-11eb-8d09-b180d0778ca1.mp4

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
for paid tickets
Screenshot from 2021-02-17 08-59-12

**Review by iamareebjamal — DISMISSED** [review]

**Comment by iamareebjamal** [comment]
Good job

File: app/templates/components/orders/order-summary.hbs [hunk 1]
```diff
@@ -67,60 +67,66 @@
...
-                  {{#if this.event.tax.isTaxIncludedInPrice}}
-                    {{t 'The price is inclusive of all taxes.'}}
-                  {{else}}
-                    {{#if ticket.price}}
-                      {{this.event.tax.rate}}%
-                    {{else}}
-                      {{t 'N/A'}}
-                    {{/if}}
+          {{#each-in (group-by 'name' (map-by 'ticket' this.data.attendees)) as |name tickets|}}
+            {{#if (eq ticket.name name)}}
```
Comment (iamareebjamal) [inline app/templates/components/orders/order-summary.hbs:71]: Try group by ID instead. That'd be more accurate

**Comment by mariobehling** [comment]
Seems to be working. Nice job!

**Pull request merged**